Podfile.lock

```diff
@@ -0,0 +1,34 @@
+PODS:
```

Comment:
Remove this file from source control and add to gitignore.

Reply:
removed, thanks